lib/css/modules/_subredditManager.scss

```diff
@@ -205,6 +204,7 @@
 		overflow-y: auto;
 		max-width: 300px;
 		max-height: 500px;
+		border-bottom: 1px rgba(0, 0, 0, .5) solid;
```

Comment:
Sorry to throw you another curveball, but will this end up with a double border (ul & dropdown container) if the [dropdownEditButton](https://www.reddit.com/#!settings/subredditManager/dropdownEditButton) option is turned off? Could mitigate it by adding `bodyClass: true` to the `dropdownEditButton` option config and wrapping this `border-bottom` property in `.res-subredditManager-dropdownEditButton & { ... }`